database/migrations/2019_11_19_020748_create_absent_requests_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateAbsentRequestsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('absent_requests', function (Blueprint $table) {
```

Comment:
`leave_requests` or `absent_requests`?

Reply:
I will use absent_requests but i forgot to edit the database diagram.